fe/fe-core/src/main/java/com/starrocks/statistic/StatisticsCollectJobFactory.java

```diff
@@ -107,6 +109,16 @@ private static void createJob(List<StatisticsCollectJob> allTableJobMap, Analyze
             return;
         }
 
+        // check job exclude db.table
+        String regex = job.getProperties().getOrDefault(StatsConstants.STATISTIC_EXCLUDE_PATTERN, null);
+        if (StringUtils.isNotBlank(regex)) {
+            Pattern checkRegex = Pattern.compile(regex);
+            String name = db.getFullName() + "." + table.getName();
+            if (checkRegex.matcher(name).find()) {
+                return;
```

Comment:
Is case insensitive better or not?

Reply:
I think case sensitive is better？more strick